NAVRH_ZMEN.txt

```diff
@@ -0,0 +1,37 @@
+# **********************************************************
+# REFACTOR ecb_rates
+# **********************************************************
+# 
+# 1. Ukázkový kód z README souboru na mém stroji nechodí,
+# zjistit proč, napsat testy a napravit.
```

Comment:
skusil som:
```
$ git clone git@github.com:ucetnictvi-on-line/ecb_rates.git

$ cd ecb_rates

$ bundle

$ bin/console

2.1.7 :001 > EcbRates.exchange_rate('JPY')
 => 113.59
2.1.7 :002 > EcbRates.exchange_rate('JPY', Date.today)
 => 113.59
```

urcite specy tomu neuskodia, neviem cim to mohlo byt, ze si to nerozbehol...